.github/workflows/ci.yaml

```
@@ -3,6 +3,21 @@ on:
   push:
     branches:
       - master
+    paths:
+      - .github/workflows/ci.yaml
+      - augur/
+      - docs/
```

```suggestion
      - docs/
      - scripts/
```
scripts are called from cram tests, so cram tests can fail if scripts change

---

woops missed it further down